packages/renderer/src/navigation.ts

```
@@ -31,144 +31,96 @@ import { router } from 'tinro';
     case NavigationPage.CONTAINER_EXPORT:
-      router.goto(`/containers/${request.parameters.id}/export`);
-      break;
+      return `/containers/${request.parameters.id}/export`;
     case NavigationPage.CONTAINER:
-      router.goto(`/containers/${request.parameters.id}/`);
-      break;
+      return `/containers/${request.parameters.id}/`;
     case NavigationPage.EXISTING_IMAGE_CREATE_CONTAINER:
-      router.goto(`/images/existing-image-create-container`);
-      break;
+      return `/images/existing-image-create-container`;
```

Biome (2.5.1)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)